compose/README.md

```diff
@@ -225,6 +241,15 @@ When initialization is complete, this container notifies the galaxy handlers to
 This container runs the actual webhandler.
 As of now, this container also runs nginx, job handlers, cron, docker.
 This container will wait until it is notified via the lock on /export/.initdone. You can disable this mechanism by setting NONUSE=sleeplock.
+
+To configure the container with HTTPS using certificates of your own, you need to modify docker-compose.yml to set the following environment variables
+- USE_HTTPS=True
```

Comment:
I'm wondering if we can do this with the .env files and not touch the compose file at all.
e.g. by including `- USE_HTTPS=${USE_HTTPS:-False}` etc ...

Reply:
That would be ideal but not possible to set **ports** though for a specific service. **.env** provides variables "global" for all the containers, right?

Reply:
No, env provides Variables globally to the compose file. Only things that you pass into the container in the compose file will be passed inside the container. Does this make sense?